### PR TITLE
chore(update-server): Consolidate tests

### PR DESCRIPTION
**Pull request by sotaoverride**
This consolidates `test-update` for openembedded and buildroot. 

The array of tuples packing/unpacking gymnastics takes care of avoiding pytests combinatoric invocation for params/fixtures.

### CODE REVIEW

File: update-server/tests/common/conftest.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        config_file_override=otupdate_config,
+        name_override="opentrons-test",
+        boot_id_override="dummy-boot-id-abc123",
+        loop=loop,
+    )
+    client = await loop.create_task(aiohttp_client(app))
+    return client
+
+
+br_files = dict(
```
Comment (amitlissack) [inline update-server/tests/common/conftest.py]: `br_files` and `oe_files` are the same aren't they? What's the point of testing against them?

File: update-server/tests/common/conftest.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+from otupdate import buildroot
+
+from otupdate import openembedded
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+one_up = os.path.abspath(os.path.join(__file__, "../../"))
+
+
+@pytest.fixture(params=[openembedded, buildroot])
+async def test_cli(aiohttp_client, loop, otupdate_config, request):
```
Comment (amitlissack) [inline update-server/tests/common/conftest.py:23]: This fixture is the same as the one in `consolidated-tests` isn't it?

What is the benefit of having a separate `consolidated-tests` and `common` directory?

**Review by amitlissack — COMMENTED** [review]
Good stuff. 

Just a couple questions.

File: update-server/tests/common/conftest.py (hunk 1 above)
Comment (sotaoverride) [reply update-server/tests/common/conftest.py]: there is a slight difference but `downloaded_update_file_consolidated` and `extracted_update_file_consolidated` fixtures take care of those differences, so I'm taking `br_files` and `oe_files` out

File: update-server/tests/common/conftest.py (hunk 2 above)
Comment (sotaoverride) [reply update-server/tests/common/conftest.py:23]: moving test_update.py under common and removing consolidated-tests dir

**Review by amitlissack — APPROVED** [review]
Thank you

**Pull request merged**